docs/devices/IH012-RT01.md

```diff
@@ -20,10 +20,13 @@ pageClass: device-page
 | Description | Motion sensor |
 | Exposes | occupancy, battery_low, tamper, sensitivity, keep_time, linkquality |
 | Picture | ![TuYa IH012-RT01](https://www.zigbee2mqtt.io/images/devices/IH012-RT01.jpg) |
-
+| Front PCB |  <img src="https://user-images.githubusercontent.com/816497/190694909-bbe25685-e5f1-4719-9005-3e8ea0fb3442.jpeg" width="150">
+| Back PCB |  <img src="https://user-images.githubusercontent.com/816497/190699395-7c3c4ad3-39a5-4a6a-9de9-33d89dc5cffc.jpeg" width="150">
```

Comment:
Please move this to the "Notes" section, the part you edited is automatically generated and  will be overridden on next generation run.